docs/getting_started.rst

```diff
@@ -160,11 +164,13 @@ Install Node.js, Yarn and other dependencies
 
 The Python project-specific dependencies installed above will install ``nodeenv``, which is a useful tool for using specific versions of Node.js and other Node.js tools in Python environments. To setup Node.js and Yarn within the Kolibri project environment, ensure your Python virtual environment is active, then run:
 
+For a more detailed guide to using nodeenv see :doc:`/howtos/nodeenv`.
+
 .. code-block:: bash
 
   # node.js, npm, and yarn
   nodeenv -p --node=16.13.2
-  npm install -g yarn
+  npm install --location=global yarn
 
   # other required project dependencies
   yarn install
```

Comment:
And this is where I get stuck 😕 

```
(kolibri-py3.9) kolibrien@kolibriU20:~/LE/kolibri$ nodeenv -p --node=16.16.0
 * Install prebuilt node (16.16.0) ..... done.
(kolibri-py3.9) kolibrien@kolibriU20:~/LE/kolibri$ npm install --location=global yarn

added 1 package, and audited 2 packages in 2s

found 0 vulnerabilities
(kolibri-py3.9) kolibrien@kolibriU20:~/LE/kolibri$ yarn install

Command 'yarn' not found, but can be installed with:

sudo apt install cmdtest

(kolibri-py3.9) kolibrien@kolibriU20:~/LE/kolibri$
```
[Yarn installation page](https://yarnpkg.com/getting-started/install) is not very helpful, I could not make head or tails of the whole **Corepack** business...

Reply:
@radinamatic ran through it again using `npm install -g yarn` (instead of the supposedly preferred `npm install --location=global yarn` and it worked that time, but maybe it's the reloading the venv that helps?

Reply:
Reloading the venv and restarting the VM did not work in my case  `npm install --location=global yarn` still gave the same result.